<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lbl_spaceName">
     <property name="text">
      <string>Space Name</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_switchToSpace">
     <property name="toolTip">
      <string>Switches to this space.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/EnableAndSwitchToSpace.png</normaloff>resources/icons/EnableAndSwitchToSpace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_enableSpace">
     <property name="toolTip">
      <string>Sets this space's attribute to 1 but doesn't explicitcly switch to it. Can be useful for setting keyframes.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/EnableSpace.png</normaloff>resources/icons/EnableSpace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_disableSpace">
     <property name="toolTip">
      <string>Sets this space's attribute to 0 but doesn't explicitcly switch from it in any way. Can be useful for setting keyframes.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/DisableSpace.png</normaloff>resources/icons/DisableSpace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_matchAndSwitchSpaceToControl">
     <property name="toolTip">
      <string>Matches the space to the control so the space is relative to the current control's transform. Also switches to this space.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/MatchAndSwitchSpaceToControl.png</normaloff>resources/icons/MatchAndSwitchSpaceToControl.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_matchSpaceToControl">
     <property name="toolTip">
      <string>Matches the space to the control so the space is relative to the current control's transform.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/MatchSpaceToControl.png</normaloff>resources/icons/MatchSpaceToControl.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_matchSpaceToSpace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Matches this space object to a different space. Useful if you're about to transition from that space to this space.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/MatchSpaceToSpace.png</normaloff>resources/icons/MatchSpaceToSpace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_matchAndSwitchControlToSpace">
     <property name="toolTip">
      <string>Match the control to the space and switch to that space. Teleports the control's transform such that its current transform snaps to the new space.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/MatchControlToSpaceAndSwitch.png</normaloff>resources/icons/MatchControlToSpaceAndSwitch.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_matchControlToSpace">
     <property name="toolTip">
      <string>Match the control to the space. Teleports the control's transform such that its current transform snaps to the new space. Doesn't switch to the space so you should expect to visually see your control snap to a different transform until you switch to that space.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/MatchControlToSpace.png</normaloff>resources/icons/MatchControlToSpace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_selectSpaceObject">
     <property name="toolTip">
      <string>Selects the space object that the control is relative to.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/SelectSpaceObject.png</normaloff>resources/icons/SelectSpaceObject.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_zeroSpaceObject">
     <property name="toolTip">
      <string>Zeroes out the space object transform, assuming it'll be back in its original spot.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/icons/ZeroSpaceObject.png</normaloff>resources/icons/ZeroSpaceObject.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
